config/schemas/config.yaml

```diff
@@ -7409,6 +7409,60 @@ properties:
         examples:
           - default-timeout-minutes: "30"
         type: object
+  gpu:
+    title: GPU Config
+    description: Configure the GPU Operator and its dependencies
+    type: object
+    properties:
+      enabled:
+        title: GPU Operator Enabled
+        default: false
+        type: boolean
+      operator:
+        title: GPU Operator Config
+        description: Configure GPU Operator
+        type: object
+        properties:
+          resources:
+            $ref: '#/$defs/kubernetesResourceRequirements'
+          tolerations:
+            $ref: '#/$defs/kubernetesTolerations'
+          affinity:
+            $ref: '#/$defs/io.k8s.api.core.v1.Affinity'
+      nodeFeatureDiscovery:
+        title: Node Feature Discovery Config
+        description: Configure Node Feature Discovery
+        type: object
+        properties:
+          master:
+            title: Node Feature Discovery master Config
+            description: Configure Node Feature Discovery masters
```

Comment:
Would have rather seen this be called `controlPlane`.

Reply:
Sure. I updated it.